graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -104,13 +104,154 @@ public void upgrade() {
                         "   - name: Security"
         );
         ensureCollector(
-                "nxlog",
+                "nxlog-linux",
                 "exec",
                 "linux",
-                "/usr/lib/graylog-sidecar/nxlog",
+                "/usr/bin/nxlog",
                 "-f -c %s",
                 "-v -c %s",
-                ""
+                "define ROOT /usr/bin\n" +
+                        "\n" +
+                        "<Extension gelfExt>\n" +
+                        "  Module xm_gelf\n" +
+                        "  # Avoid truncation of the short_message field to 64 characters.\n" +
+                        "  ShortMessageLength 65536\n" +
+                        "</Extension>\n" +
+                        "\n" +
+                        "<Extension syslogExt>\n" +
+                        "  Module xm_syslog\n" +
+                        "</Extension>\n" +
+                        "\n" +
+                        "User nxlog\n" +
+                        "Group nxlog\n" +
+                        "\n" +
+                        "Moduledir /usr/lib/nxlog/modules\n" +
+                        "CacheDir /var/spool/collector-sidecar/nxlog\n" +
+                        "PidFile /var/run/graylog-sidecar/nxlog.pid\n" +
+                        "LogFile /var/log/graylog-sidecar/nxlog.log\n" +
+                        "LogLevel INFO\n" +
+                        "\n" +
+                        "\n" +
+                        "<Input file>\n" +
+                        "\tModule im_file\n" +
+                        "\tFile '/var/log/*.log'\n" +
+                        "\tPollInterval 1\n" +
+                        "\tSavePos\tTrue\n" +
+                        "\tReadFromLast True\n" +
+                        "\tRecursive False\n" +
+                        "\tRenameCheck False\n" +
+                        "\tExec $FileName = file_name(); # Send file name with each message\n" +
+                        "</Input>\n" +
+                        "\n" +
+                        "<Input syslog-udp>\n" +
+                        "\tModule im_udp\n" +
+                        "\tHost 127.0.0.1\n" +
+                        "\tPort 514\n" +
```

Comment:
I wouldn't start a syslog input by default. We should either use an unprivileged port for this or comment the input if we really want to show how this works.

Reply:
Ack. I'll comment this out